test/integration/server.h

```diff
@@ -430,10 +430,11 @@ class IntegrationTestServer : public Logger::Loggable<Logger::Id::testing>,
   void onWorkersStarted() override {}
 
   void start(const Network::Address::IpVersion version,
-             std::function<void()> on_server_init_function, bool deterministic,
-             bool defer_listener_finalization, ProcessObjectOptRef process_object,
-             Server::FieldValidationConfig validation_config, uint32_t concurrency,
-             std::chrono::seconds drain_time, Server::DrainStrategy drain_strategy,
+             std::function<void()> on_server_init_function,
+             absl::optional<uint64_t> deterministic_value, bool defer_listener_finalization,
```

Comment:
@alyssawilk Mind giving this change to propagate a deterministic value through the integration test framework a look?

Reply:
Looks solid!  Lots of plumbing but thankfully there was that argument to replace so there wasn't a need to track APIs down from scratch :-)

I could go either way on more-accurate-naming vs churn so I'll leave that up to you.

Reply:
I assume this part is LGed then :) 

Please do reopen if anything needed to be changed. Thanks!